src/rand.rs

```diff
@@ -279,6 +284,19 @@ mod darwin {
     }
 }
 
+#[cfg(target_env = "sgx")]
+mod rdrandom {
+    use crate::error;
+    use rdrand::RdRand;
+    use rand_core::RngCore;
```

Comment:
I don't want to use `rand_core` or `rdrand` in this project. Instead, let's just use the BoringSSL rdrand code via FFI.

Please take the Cargo.toml and rand.rs changes out of this PR and let's do them in another PR.

Reply:
I'm fine with not using the external crates.

> Please take the Cargo.toml and rand.rs changes out of this PR and let's do them in another PR.

Not sure what you mean here. This is the entire PR?